scripts/get-model-data.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+"""Export a CSV of all models or something
+
+Looks for a corresponding find_{model}_iter() method in the api client and returns
+all of the results in a CSV with only requested keys being used as columns
+Before generating the final csv, it is possible to:
+- filter the list, looking for only those entries with a particular value
+- do some additional processing on a value
+- sort the list
+
+Usage:
+    scripts/get-model-data.py [-h] <stage> <api_token> <model> <output_dir> [--limit=<limit>][--verbose]
+
+Options:
+    -h --help
+"""
+import sys
+sys.path.insert(0, '.')
+
+from docopt import docopt
+from dmapiclient import DataAPIClient
+from dmscripts.env import get_api_endpoint_from_stage
+from models.modeltrawler import ModelTrawler
+from models.utils import (
+    process_collection, return_filtered_collection, return_sorted_collection,
+    format_datetime_string_as_date, remove_username_from_email_address
+)
+from models.writecsv import write_csv
+
+CONFIGS = {
+    'buyers': {
+        'base_model': 'users',
+        'keys': ('id', 'createdAt', 'role'),
+        'get_data_kwargs': {'print_counts': 100},
+        'process_rules': {
+            'createdAt': format_datetime_string_as_date
+        },
+        'filter_rules': [
+            ('role', '==', 'buyer'),
```

Comment:
We could use lambdas here directly, but if the plan is to transition to pandas query language soon, then we can keep this as is